src/index.tsx

```diff
@@ -66,11 +77,14 @@ const renderIndex = async () => {
     <Sentry.ErrorBoundary>
       <ConfigContext.Provider value={config}>
         <Provider store={store}>
-          <EnkiIntlProvider>
+          <SandboxFeature
```

Comment:
I see there's an improvement to be made to the SandboxFeature component. the renderFallback should allow to pass props, so that it can take children, that way we don't have to repeat them inside "EnkiApp".

Reply:
It actually worked with children 😄 I just moved it away to make it look shorter

Reply:
With the feature flag off, no problems

Reply:
With feature flag set to false, it does produce this kinds of thing (same as with feature on) when editing a form input on any page
```
@sentry_react.js?v=bca3daac:2942 Warning: A component is changing an uncontrolled input to be controlled. This is likely caused by the value changing from undefined to a defined value, which should not happen. Decide between using a controlled or uncontrolled input element for the lifetime of the component. More info: https://reactjs.org/link/controlled-components
    at input
    at http://localhost:3001/node_modules/.vite/deps/chunk-57OQC4YP.js?v=fa3f8736:1084:24
    at div
    at InputGroupContextProvider2 (http://localhost:3001/node_modules/.vite/deps/chunk-57OQC4YP.js?v=fa3f8736:649:23)
    at http://localhost:3001/node_modules/.vite/deps/chunk-57OQC4YP.js?v=fa3f8736:691:23
    at http://localhost:3001/node_modules/.vite/deps/chunk-57OQC4YP.js?v=fa3f8736:1051:22
    at section
    at div
    at window.$RefreshReg$ (http://localhost:3001/src/components/GeneralLineEditor/index.tsx:36:3)
    at section
    at LineEditorSteps (http://localhost:3001/src/scenes/LineEditor/LineEditorSteps.tsx:24:11)
    at div
    at div
    at OverlayLoader (http://localhost:3001/src/components/OverlayLoader/index.tsx:22:3)
    at div
    at window.$RefreshReg$ (http://localhost:3001/src/components/LineEditorStepper/index.tsx:30:3)
    at Loading (http://localhost:3001/src/components/Loading/index.tsx:21:3)
    at div
    at div
    at Page (http://localhost:3001/src/components/Page/index.tsx:24:3)
    at commitMutationEffectsOnFiber (http://localhost:3001/node_modules/.vite/deps/chunk-WBIQBQPZ.js?v=fa3f8736:17699:15)
    at RenderedRoute (http://localhost:3001/node_modules/.vite/deps/chunk-ZZJRKN7J.js?v=fa3f8736:3630:5)
    at Routes (http://localhost:3001/node_modules/.vite/deps/chunk-ZZJRKN7J.js?v=fa3f8736:4134:5)
    at div
    at commitMutationEffectsOnFiber (http://localhost:3001/node_modules/.vite/deps/chunk-WBIQBQPZ.js?v=fa3f8736:17699:15)
    at Loading (http://localhost:3001/src/components/Loading/index.tsx:21:3)
    at div
    at div
    at div
    at ScrollToTop (http://localhost:3001/src/components/ScrollToTop/index.tsx:20:24)
    at Router (http://localhost:3001/node_modules/.vite/deps/chunk-ZZJRKN7J.js?v=fa3f8736:4077:15)
    at BrowserRouter (http://localhost:3001/node_modules/.vite/deps/react-router-dom.js?v=0683f181:549:5)
    at div
    at commitMutationEffectsOnFiber (http://localhost:3001/node_modules/.vite/deps/chunk-WBIQBQPZ.js?v=fa3f8736:17699:15)
    at ApolloProvider (http://localhost:3001/node_modules/.vite/deps/@apollo_client.js?v=a8152831:6238:19)
    at Apollo (http://localhost:3001/src/api/index.tsx:69:26)
    at commitMutationEffectsOnFiber (http://localhost:3001/node_modules/.vite/deps/chunk-WBIQBQPZ.js?v=fa3f8736:17699:15)
    at AuthProvider (http://localhost:3001/node_modules/.vite/deps/react-oidc-context.js?v=572331a2:3188:5)
    at AuthProvider (http://localhost:3001/src/auth/auth.tsx:106:32)
    at commitMutationEffectsOnFiber (http://localhost:3001/node_modules/.vite/deps/chunk-WBIQBQPZ.js?v=fa3f8736:17699:15)
    at Suspense
    at EnkiIntlProvider (http://localhost:3001/src/i18n/EnkiIntlProvider.tsx:25:3)
    at EnkiApp
    at Suspense
    at SandboxFeature (http://localhost:3001/src/ext/SandboxFeature.tsx:21:3)
    at Provider (http://localhost:3001/node_modules/.vite/deps/react-redux.js?v=a7105bce:1253:3)
    at commitMutationEffectsOnFiber (http://localhost:3001/node_modules/.vite/deps/chunk-WBIQBQPZ.js?v=fa3f8736:17699:15)
```

Reply:
I think this warning is unrelated

Reply:
But the stuff above is actually reproducing in the master branch as well, so at least not related to these changes